modules/aws-waf/README.md

```diff
@@ -0,0 +1,23 @@
+# AWS WAF
```

Comment:
Did you generate this file or wrote it manually? It does not correspond to source files, as you have to put such comment in `main.tf` for `./scripts/generate-docs.sh` to work correctly:
```tf
/**
 * # AWS WAF
 * 
 * This module is used to restrict access to a CloudFront distribution based on IP whitelist.
 * It creates a Web Application Firewall rule that can be added to an existing CF distribution.
 */
```